docs/index.md

```diff
@@ -2276,6 +2276,12 @@ certain functions of the mmj2 tool:
   must be used as the justification. You can do the same by
   selecting it as the "Label".
 
+#### Unification
+
+Metamath-lamp's current unification algoritm is only unidirectional, that is, it can only substitute in one direction to find a match. In the technical literature this "unification" algorithm is often called "matching" instead of being considered full syntactic unification. Thiz is contrast to ither tools, like mmj2, which implement full syntactic unification.
+
+There are [some discussions about removing this limitation in metamath-lamp](https://github.com/expln/metamath-lamp/issues/77). There's nothing fundamental to metamath-lamp about this limitation; the issue is that it takes time to write a correct and performant implementation of unification. We hope rhat future versions of the tool will remove this limitation. However, current users must work around this limitation.
+
```

Comment:
Should I claim there's an intent to eventually update to the full unification algorithm, at least as an option?

Reply:
If so, it's probably a good addition to "current limitations" at the top like undo/redo. Like undo/redo, implementing full unification so quite doable, it just takes time and thus has not been done yet.

Reply:
Yes, it makes sense to mention this limitation and write that it will be addressed in the future.

One suggestion, I would not mention "correct" in the following sentence: `it takes time to write a correct and performant implementation of unification`. Instead, I would write `it takes time to write a performant implementation of full unification`. Mentioning "correct" here makes an impression (as for me) that current implementation of mm-lamp algorithms is incorrect. But as I know they are correct (or to be precise there is no evidence they are incorrect so far), just not so powerful as in mmj2.

Reply:
Good point, I didn't mean the current one was *incorrect*, just that writing a changed algorithm takes time if you want the new one to be correct as well. I'll change it! Since you intend to eventually add that, I'll add full syntactic unification to the list along with undo/redo of examples of things that "the tool is expected to eventually do but doesn't do yet".

Reply:
Thank you!